.tool-versions

```diff
@@ -1,4 +1,4 @@
-elixir 1.13.3-otp-24
-erlang 24.3.2
+elixir 1.13.4-otp-24
+erlang 24.3.4.4
```

Comment:
it looks like there's actually 24.3.4.5 out as of today it looks like not sure if you want to go ahead and bump to that or not.

Reply:
Sure thing.